app/models/service_ansible_playbook.rb

```
@@ -73,4 +90,29 @@ def extra_vars_from_dialog
 
     params.blank? ? {} : {:extra_vars => params}
   end
+
+  def create_inventory(action, hosts)
+    manager(action).with_provider_connection do |connection|
+      inventory = connection.api.inventories.create(:name => inventory_name(action), :organization => 1)
```

Would we want `.create!` and `destroy!` - so we could bubble up an Exception?